### PR TITLE
start spanish translation pages + french translation signs pages

### PR DESCRIPTION
**Pull request by ObjectifMoon21**
Spanish translation of the pages:

`404_es.json`
`bank-runs_es.json`
`bitcoin-vs-cbdc_es.json`
`bitcoin-vs-gold_es.json`

--------------------------------------

French translation of new strings: 

`i18n\fr\inflation_fr.json`

French translation of signs pages:

`i18n\fr\sign-success_fr.json`
`i18n\fr\signs_fr.json`

### CODE REVIEW

**Comment by ObjectifMoon21** [comment]
Hi!
Today i translated these pages in Spanish:

- i18n\es\common_es.json
- i18n\es\compound-inflation-calculator_es.json
- i18n\es\get-involved_es.json
- i18n\es\index_es.json
- i18n\es\postcard-success_es.json
- i18n\es\postcards_es.json
- i18n\es\sticker-language-success_es.json
- i18n\es\sticker-success_es.json
- i18n\es\stickers_es.json

I'm not sure if they are automatically added to this PR. Please inform me if i need to create another PR for the new pages.

**Comment by ObjectifMoon21** [comment]
More pages translated in Spanish today:

- i18n\es\inflation_es.json
- i18n\es\lightning_es.json
- i18n\es\wallets_es.json

File: i18n/es/common_es.json
Comment (sovenor) [inline i18n/es/common_es.json:1]: Are there better translations for COLD WALLET and HOT WALLET at the bottom of this file? @ObjectifMoon21

**Review by sovenor — APPROVED** [review]
This looks great, thank you! I will work on enabling Spanish translations on the website now. Thanks for spending so much time on this!

**Pull request merged**